modules/telco-core-security.adoc

```diff
@@ -1,31 +1,68 @@
 // Module included in the following assemblies:
 //
-// * scalability_and_performance/telco_ref_design_specs/core/telco-core-ref-design-components.adoc
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
 
 :_mod-docs-content-type: REFERENCE
 [id="telco-core-security_{context}"]
 = Security
 
 New in this release::
-//CNF-11806
-* Secure boot host firmware setting is now recommended for telco core clusters.
-For more information, see "Host firmware and boot loader configuration".
+// https://issues.redhat.com/browse/CNF-13768
+* link:https://access.redhat.com/articles/7090422[New knowledgebase article on creating custom node firewall rules]
 
 Description::
-You should harden clusters against multiple attack vectors.
++
+--
+Telco customers are security conscious and require clusters to be hardened against multiple attack vectors.
 In {product-title}, there is no single component or feature responsible for securing a cluster.
 Use the following security-oriented features and configurations to secure your clusters:
 
 * **SecurityContextConstraints (SCC)**: All workload pods should be run with `restricted-v2` or `restricted` SCC.
-* **Seccomp**: All pods should be run with the `RuntimeDefault` (or stronger) seccomp profile.
-* **Rootless DPDK pods**: Many user-plane networking (DPDK) CNFs require pods to run with root privileges. With this feature, a conformant DPDK pod can be run without requiring root privileges.
+* **Seccomp**: All pods should run with the `RuntimeDefault` (or stronger) seccomp profile.
+* **Rootless DPDK pods**: Many user-plane networking (DPDK) CNFs require pods to run with root privileges.
+With this feature, a conformant DPDK pod can run without requiring root privileges.
 Rootless DPDK pods create a tap device in a rootless pod that injects traffic from a DPDK application to the kernel.
-* **Storage**: The storage network should be isolated and non-routable to other cluster networks. See the "Storage" section for additional details.
+* **Storage**: The storage network should be isolated and non-routable to other cluster networks.
+See the "Storage" section for additional details.
+
+Refer to link:https://access.redhat.com/articles/7090422[Custom nftable firewall rules in OpenShift] for a supported method of implementing custom nftables firewall rules in OpenShift cluster nodes.
```

Comment:
I notice quite a few instances of "OpenShift" in this module. Can we say "OpenShift" cluster, "OpenShift environments" etc., or should we use the {product-title} attribute?